common/projectconfig/projectconfig.go

```diff
@@ -32,14 +32,16 @@ type Config struct {
 	ExternalDirs  []string                    `toml:"external-dirs"`
 	Commands      Commands                    `toml:"commands"`
 	FTLMinVersion string                      `toml:"ftl-min-version"`
+	absModuleDirs []string
 }
 
-// ModuleDirsOrDefault returns the module-dirs field from the ftl-project.toml, unless
+// AbsModuleDirsOrDefault returns the absolute path for the module-dirs field from the ftl-project.toml, unless
 // that is not defined, in which case it defaults to the root directory.
-func (c Config) ModuleDirsOrDefault() []string {
-	if len(c.ModuleDirs) > 0 {
-		return c.ModuleDirs
+func (c Config) AbsModuleDirsOrDefault() []string {
+	if len(c.absModuleDirs) > 0 {
+		return c.absModuleDirs
 	}
+
 	return []string{"."}
```

Comment:
Since `"."` is not an absolute path, do you want to update that to be absolute in this function as well?

Reply:
Yeah good question! Maybe we should just add this absolute path when parsing/merge if there's nothing in the `toml` file. Then this would just return the `absModuleDirs` instead of `"."`

Reply:
That makes sense to me :)